obspy/core/util/testing.py

```diff
@@ -729,6 +729,7 @@ def _warn_explicit(self, message, category, *args, **kwargs):
                                     filename="", lineno=0))
 
     def _warn(self, message, category=None, *args, **kwargs):
+        category = Warning if category is None else category
```

Comment:
Any reason to not do `def _warn(self, message, category=Warning, *args, **kwargs):`?

Reply:
Mainly that the class definition is mutable and it can be better to avoid mutable default args. Probably wouldn't hurt anything here though.

Reply:
ok, changed it.

Reply:
Ya I think it will be fine in this case, and it does make the code a bit cleaner.